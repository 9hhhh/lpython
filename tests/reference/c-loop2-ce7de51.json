{
    "basename": "c-loop2-ce7de51",
    "cmd": "lpython --no-color --show-c {infile}",
    "infile": "tests/loop2.py",
    "infile_hash": "7946c522ceb16f99810780d4aba7fa2593695a4b49fb35ea1f131f53",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "c-loop2-ce7de51.stdout",
    "stdout_hash": "06b2775b834c9216cb15a9fa6104d895c16b36e92ccece3da7b90576",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}