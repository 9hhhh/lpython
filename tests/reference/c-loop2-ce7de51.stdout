#include <assert.h>
#include <complex.h>
#include <inttypes.h>
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include <stdbool.h>

#include <lfortran_intrinsics.h>

// Forward declarations
void _lpython_main_program();
void test_for();
void _xx_lcompilers_changed_exit_xx(int32_t error_code);

// Implementations
void _lpython_main_program()
{
    test_for();
}

void test_for()
{
    int32_t i;
    for (i=0; i<=10 - 1; i++) {
        if (i == 0) {
            continue;
        }
        if (i > 5) {
            break;
        }
        if (i == 3) {
            exit(0);
        }
    }
    _xx_lcompilers_changed_exit_xx(2);
}

void _xx_lcompilers_changed_exit_xx(int32_t error_code)
{
    exit(error_code);
}

int main(int argc, char* argv[])
{
    _lpython_main_program();
    return 0;
}
